package com.zhiyiyo.crm;

import org.junit.Test;

public class TestException {
    @Test
    public void testGetCause(){
        String a="123";
        System.out.println(a.compareTo(""));
    }
}
